js/reducers/async-requests.js

```diff
@@ -1,3 +1,5 @@
+/* eslint brace-style: 0 */
```

Comment:
dropping these in favor of consistent styles. might be a bit ugly, but that's okay